### PR TITLE
Feature/new swanky docs page

### PR DESCRIPTION
**Pull request by codespool**

### CODE REVIEW

**Comment by codespool** [comment]
Sorry, I changed my username and email in git config when I was creating those gifs, and forgot to switch them back, so the commits are unverified

**Review by fiexer — APPROVED** [review]

**Comment by fiexer** [comment]
LGTM

File: docs/wasm/sc-dev/swanky.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+
+<Figure caption="Deploy a contract" src={require('./img/08-deploy.gif').default} />
+
+## Swanky Node
+
+Swanky node is a Substrate based blockchain configured to enable `pallet-contracts` (a smart contract module) and more features to help Wasm smart contract development locally.
+
+### Features
+
+- [`pallet-contracts`] (polkadot-0.9.19) and its unstable-feature are enabled by default.
```
Comment (PierreOssun) [inline docs/wasm/sc-dev/swanky.md:207]: Please update this part & Compatible ink! version

File: docs/wasm/sc-dev/swanky.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+For more detailed documentation please visit the [`swanky-node` GitHub repo].
+
+## Documentation
+
+[`swanky` CLI Github repo] with the latest documentation
+
+[`swanky-node` Github repo] with the latest documentation.
+
+[`pallet-contracts`]: https://github.com/paritytech/substrate/tree/master/frame/contracts
+[`pallet-dapps-staking`]: https://github.com/AstarNetwork/astar-frame/tree/polkadot-v0.9.19/frame/dapps-staking
```
Comment (PierreOssun) [inline docs/wasm/sc-dev/swanky.md:299]: update links

**Pull request merged**